xml/rmt_mirroring.xml

```diff
@@ -455,6 +455,12 @@ Clean finished. An estimated 157 MB were removed.
     </para>
     <screen>&prompt.root.sun;<command>umount /mnt/external</command></screen>
    </step>
+   <step>
+    <para>
+     If not yet done, set up RMT on <literal>sirius</literal> using "yast2 rmt". 
+     In case of an offline RMT setup, choose "Skip" on the "Organization Credentials" screen.
```

Comment:
```suggestion
     If not yet done, set up &rmt; on <literal>sirius</literal> by running the
     <command>yast2 rmt</command>. 
     In case of an offline &rmt; setup, select "Skip" on the <guimenu>Organization Credentials</guimenu> screen.
```